variables.tf

```
@@ -118,6 +125,12 @@ variable "gitignore_template" {
   default     = null
 }
 
+variable "is_template" {
+  description = "Whether or not to tell GitHub that this is a template repository. ( Default: false)"
+  type        = bool
+  default     = false
```

should be `null` for defaults
```suggestion
  default     = null
```

---

fixed in 84ebec5